clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -104,6 +104,12 @@ New check aliases
 Changes in existing checks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- Improved :doc:`modernize-use-std-print
+  <clang-tidy/checks/modernize/use-std-format>` and
```

Comment:
wrong name, and split that entry into 2, even if you would need to duplicate description